sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -161,6 +161,8 @@ class Analyzer(
       HandleNullInputsForUDF),
     Batch("FixNullability", Once,
       FixNullability),
+    Batch("ResolveAggAliasInGroupBy", Once,
+      ResolveAggAliasInGroupBy),
```

Comment:
yea, it seems reasonable to run that rule with this `ResolveAggAliasInGroupBy` together in one batch